quiche/src/lib.rs

```diff
@@ -1899,8 +1915,7 @@ impl Connection {
 
         conn.handshake.init(is_server)?;
 
-        conn.handshake
-            .use_legacy_codepoint(config.version != PROTOCOL_VERSION_V1);
+        conn.handshake.use_legacy_codepoint(config.version != PROTOCOL_VERSION_V1);
```

Comment:
Make sure you use cargo +nightly format when you commit changes otherwise the test suite will simply not run. For convenience I ended up using https://pre-commit.com/